client/my-sites/site-settings/style.scss

```diff
@@ -160,6 +160,26 @@
 	}
 }
 
+.site-settings__general-settings {
+	.form-toggle__label {
+		display: flex;
+		align-items: center;
+	}
+
+	.form-toggle__switch {
+		flex: 1 0 auto;
+	}
+
+	.form-toggle__switch + span {
```

Comment:
Good point, I've added a class and styled it by that class. @ryelle you might want to apply this to #10167 as well.